### PR TITLE
SAK-42072 calendar,content: ...newTime().breakdownLocal().getYear() =…

### PR DESCRIPTION
**Pull request by axxter99**
int year = TimeService.newTime().breakdownLocal().getYear() =>
int year = Year.now().getValue();

### CODE REVIEW

**Comment by axxter99** [comment]
@adrianfish Year.now(ZoneId) ?

**Comment by adrianfish** [comment]
I saw that and wondered too. It doesn't sound like it does anything other than just shift the timezone. If the thing we need is the local time then this is probably fine. Maybe put a comment on one of the now() statements stating that assumption?

File: calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarFilter.java [hunk 1]
```diff
@@ -24,7 +24,12 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.time.Year;
-import java.util.*;
+import java.time.ZonedDateTime;
+import java.util.Calendar;
+import java.util.Date;
```
Comment (adrianfish) [inline calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarFilter.java]: Can't see any use of Date in this commit. Do you need this?

File: calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarFilter.java [hunk 2]
```diff
@@ -24,7 +24,12 @@
...
+import java.util.GregorianCalendar;
+import java.util.Iterator;
```
Comment (adrianfish) [inline calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarFilter.java]: is this needed?

File: calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarFilter.java [hunk 3]
```diff
@@ -24,7 +24,12 @@
...
+import java.util.Vector;
```
Comment (adrianfish) [inline calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarFilter.java]: Do you need this ?

**Pull request closed without merging**